pkg/cmd/server/origin/ensure.go

```diff
@@ -177,6 +180,18 @@ func (c *MasterConfig) ensureComponentAuthorizationRules() {
 			glog.Errorf("Error creating bootstrap policy: %v", err)
 		}
 
+		// Wait until the policy cache has caught up before continuing
+		err := wait.Poll(100*time.Millisecond, 30*time.Second, func() (done bool, err error) {
+			if _, err := c.PolicyClient().ClusterPolicies().List(labels.Everything(), fields.Everything()); err == nil {
```

Comment:
Hmm... I guess this does an indirect check because the `PolicyClient` isn't allowed to read `ClusterPolicies` unless the authorization cache is up to date.  I'd still like a more straightforward check.